src/module/system/text-editor.ts

```diff
@@ -223,7 +223,7 @@ class TextEditorPF2e extends TextEditor {
             case "Localize":
                 return this.#localize(paramString, options);
             case "Template":
-                return this.#createTemplate(paramString, inlineLabel, item?.system);
+                return this.#createTemplate(paramString, inlineLabel, item, item?.system);
```

Comment:
If you're passing the whole item, you needn't additionally pass the `system` property.

Reply:
I wanted to change the least amount of code, and technically leaving it as an arg allows people to disconnect those two things. I'll keep it to a single argument.

Reply:
fixed